torch/csrc/lazy/ts_backend/ops/scalar.h

```diff
@@ -19,6 +19,16 @@ class TORCH_API Scalar : public TsNode {
   Scalar(const at::Scalar& value, Shape shape);
   Scalar(const at::Scalar& value, c10::ScalarType type);
 
+  bool CanBeReused(const at::Scalar& value, Shape shape) const {
+    TORCH_INTERNAL_ASSERT(false, "Reusing Scalar nodes is unsupported")
```

Comment:
hmm, so, i guess you actually prevent this by not calling ReuseOrMakeNode on scalar, but this is just a backup in case someone tried to change that?